lms/static/js/student_account/views/FormView.js

```diff
@@ -163,12 +167,29 @@
                     }
 
                     if (key) {
+                        if (this.interesting_fields($el)) {
+                            this.remove_validation_error($el, $form);
```

Comment:
I believe `FormView` is generic for all views like register, login and reset password, could we move all the logic added in this file into `LoginView`?

Reply:
This functionality suppose to work for both login/reset form that is why it is placed in FormView.